x-pack/packages/kbn-elastic-assistant/impl/assistant/chat_send/use_chat_send.tsx

```diff
@@ -97,7 +98,17 @@ export const useChatSend = ({
         selectedSystemPrompt: systemPrompt,
       });
 
-      const replacements = userMessage.replacements ?? currentConversation.replacements;
+      const baseReplacements: Replacements =
```

Comment:
Is there actually a case where replacements are on the `userMessage`?

Reply:
Yes, it appears that when `getCombinedMessage()` creates the `userMessage`, some paths in `getCombinedMessage()` return `replacements`. In `main` the `userMessage.replacements` are referenced [here](https://github.com/elastic/kibana/blob/943dfea82cb55f121262c057f9ee585c515ad91c/x-pack/packages/kbn-elastic-assistant/impl/assistant/chat_send/use_chat_send.tsx#L100) in the following code:

```ts
const replacements = userMessage.replacements ?? currentConversation.replacements;
```

, which appears to be introduced as part of https://github.com/elastic/kibana/pull/173487

Reply:
Got it, thank you!